package.json

```diff
@@ -5,8 +5,10 @@
   "scripts": {
     "start": "rm -rf ./node_modules/.cache/babel-loader && node scripts/start.js",
     "build": "node scripts/build.js",
-    "test": "npm run test:lint && node scripts/test.js",
-    "test:lint": "standard"
+    "test": "npm run test:lint && npm run test:unit",
+    "test:lint": "standard",
+    "test:unit": "node scripts/test.js",
+    "ci": "CI=1 npm run test:unit"
```

Comment:
what does setting CI=1 do?

Reply:
Disables watch mode ;)
https://github.com/Holo-Host/hp-admin/blob/develop/scripts/test.js#L40

Reply:
BTW, looks to me there's no real reason to use test.js instead of simple npm script...